streams/src/test/java/org/apache/kafka/streams/kstream/internals/suppress/KTableSuppressProcessorTest.java

```diff
@@ -60,13 +62,36 @@
 
     private static final Change<Long> ARBITRARY_CHANGE = new Change<>(7L, 14L);
 
+    private static class Harness<K, V> {
+        private final KTableSuppressProcessor<K, V> processor;
+        private final MockInternalProcessorContext context;
+
+
+        Harness(final Suppressed<K> suppressed,
+                final Serde<K> keySerde,
+                final Serde<V> valueSerde) {
+
+            final String storeName = "test-store";
+
+            final StateStore buffer = new InMemoryTimeOrderedKeyValueBufferStore.Builder(storeName).build();
+            final KTableSuppressProcessor<K, V> processor =
+                new KTableSuppressProcessor<>(getImpl(suppressed), storeName, keySerde, new FullChangeSerde<>(valueSerde));
+
+            final MockInternalProcessorContext context = new MockInternalProcessorContext();
+            buffer.init(context, buffer);
+            processor.init(context);
+
+            this.processor = processor;
+            this.context = context;
+        }
+    }
```

Comment:
It's a little bit of orchestration to create the processor, the context, and the store, initialize everything, and return the processor and the context, so I bundled it all up here.